verilog/tools/kythe/kythe_facts_extractor.cc

```diff
@@ -1058,6 +1058,18 @@ void KytheFactsExtractor::ReferenceExtendsInheritance(
     return;
   }
 
+  // TODO(hzeller): should this have been detected before ?
+  // NULL vname is not encountered, but NULL scope. Issue #1128
+  if (!definitions.back().first) {
+    LOG(ERROR) << "ReferenceExtendsInheritance: NULL vname";
+    return;
+  }
+  if (!definitions.back().second) {
+    LOG(ERROR) << "ReferenceExtendsInheritance: NULL scope for vname "
```

Comment:
Check the lines 292 and 316. IIUC, null scope is the global scope. But this whole behavior looks buggy. Including `ScopeResolver::SearchForDefinitions` (verilog/tools/kythe/scope_resolver.cc) -- it doesn't check if the `current_scope` is null  before adding it to the `declarations` vector. But, most of the callers perfom the null-check.

IMO, this PR is OK as a workaround. But we need to revisit the handling of scopes (and clarify the semantic for the global scope).

Reply:
Yes, this is why I opened #1128 for further investigation.